shared/src/ledger/queries/shell/eth_bridge.rs

```diff
@@ -34,15 +35,52 @@ use namada_ethereum_bridge::storage::{
     bridge_contract_key, native_erc20_key, vote_tallies,
 };
 use namada_proof_of_stake::pos_queries::PosQueries;
+use serde::{Deserialize, Serialize};
 
 use crate::eth_bridge::ethers::abi::AbiDecode;
+use crate::ledger::queries::shell::EventType;
 use crate::ledger::queries::{EncodedResponseQuery, RequestCtx, RequestQuery};
 use crate::types::eth_abi::{Encode, EncodeCell};
 use crate::types::eth_bridge_pool::PendingTransfer;
 use crate::types::keccak::KeccakHash;
 use crate::types::storage::Epoch;
 use crate::types::storage::MembershipProof::BridgePool;
 
+/// Container for the status of queried transfers to Ethereum.
```

Comment:
I see, so these are silently filtered out. Do we want to say that a hash not corresponding to any transfer was input in case someone does bad copy paste?